### ISSUE TITLE
Please add telescope support

### ISSUE THREAD

**Issue by MordechaiHadad** [RishabhRD/nvim-cheat.sh#11]
Telescope support would be great for listing things.

**Comment by UltiRequiem** [comment]
This would be cool

**Comment by RishabhRD** [comment]
I agree. I need some days to implement as I am going through my undergrad job interviews. So unable to spend time on this plugin from many days.

**Comment by enribd** [comment]
Definitely a good feature.

Maybe it could be integrated as a Telescope extension -> https://github.com/nvim-telescope/telescope.nvim#extensions

**Comment by convers39** [comment]
Any updates on this feature？
